doc/RFCs/username_specifications.md

```diff
@@ -0,0 +1,84 @@
+#### _Username_ specifications for Caliopen accounts
+
+This document describes valid **username** for the creation of an account within Caliopen instances.
+
+#### Preambule
+Living in the XXI century, Caliopen's users should be able to take (almost) any username they want to. The following specifications are as open as possible.
+
+##### Definition :
+An **username** is the unique identifier an user makes use of to create an account within a Caliopen instance. 
+Username is only an identifier for the user's account : it is not necessarily the user's real name, or email, or nickname… It will only be used as a credential for the purpose of identifying the user when logging in Caliopen. The username is unique within a Caliopen instance.
+
+NB : once an user has chosen an _username_, she/he will be able to create or add some « identities », that are made of : first name, family name, email, etc.
+
+##### Format :
+ 
+* Username are made of utf-8 "characters".  
+By "character" we mean a single Unicode grapheme, encoded as a single Unicode code point.
+* Username **is** case sensitive.
```

Comment:
We've always thought that the **username** really was the local part of an email, even if the user does'nt know it at first (he may choose to use it later when he becomes aware that his other social identities have poor PIs). I'm not sure we really want to change our mind about it, as it may be too confusing for an average user to create an email specific to the Caliopen instance that would be different from his **username**.

This doesn't mean that username must me case insensitive, as local part of an email **may** be case sensitive, but AFAIK no major emailing system distinguish different addresses based on case. So I would prefer those usernames to be case insensitive anyway.